RELEASENOTES-1.4.docu

```diff
@@ -303,4 +303,12 @@
       </pre></add-note></build-id>
   <build-id value="next"><add-note>The <tt>issue</tt> method in the
       <tt>map_target</tt> template can now be overridden.</add-note></build-id>
+  <build-id value="next"><add-note> Objects of type `port` and `bank`
+      can now be placed inside `group` objects.</add-note></build-id>
+  <build-id value="next"><add-note> Added a new template
+      `subdevice_port`, which changes how the port's Simics object is
+      named. The object for `port p is subdevice_port;` is named
+      `dev.p` instead of `dev.port.p`. Also, objects of type `port`
+      and `bank` can be placed inside a `port` object that
+      instantiates `subdevice_port`.</add-note></build-id>
```

Comment:
The release notes need to be updated, this is a new object type, not template. Same thing with `RELEASENOTES-1.2`.

Reply:
done